src/ot_meter_noop.erl

```diff
@@ -19,7 +19,8 @@
 
```

Comment:
For other instruments you have used prefix names like `ot_sum_observer`, here you use suffix naming. I think one or the other should be used. I'd prefer suffix naming.

Reply:
I'm not sure what you are referencing here in the noop module?

Reply:
@tolbrino oh, I think of `ot_sum_observer` as a suffix naming. Like `noop` is a type of the `ot_meter`, `observer` is a type of sum aggregation... But I can see how it would also make sense as `ot_observer_sum`.

Hm, I think you are right, `ot_observer_*` would look better and make more sense.